src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/EnvelopeEventProcessor.java

```diff
@@ -93,10 +93,7 @@ private MessageProcessingResult process(IMessage message) {
                 return new MessageProcessingResult(UNRECOVERABLE_FAILURE, ex);
             } catch (NotificationSendingException ex) {
                 logMessageProcessingError(message, envelope, ex);
-
-                // CCD changes have been made, so it's better to dead-letter the message and
-                // not repeat them, at least until CCD operations become idempotent
-                return new MessageProcessingResult(UNRECOVERABLE_FAILURE, ex);
+                return new MessageProcessingResult(POTENTIALLY_RECOVERABLE_FAILURE, ex);
```

Comment:
I just realised I can remove this whole catch clause.

Reply:
You were adding `ex` as parameter here and it's not being added in the last `catch` clause.
Not sure what's preferrend

Reply:
Exception is only needed for `UNRECOVERABLE_FAILURE` results - the message would be in the reason in dead-lettered message. For recoverable failures we have no use for exceptions.